src/recipe.html

```diff
@@ -28,8 +19,4 @@ <h1>Make Your Recipe</h1>
       </form>
     </div>
   </main>
-  <footer id="main-footer"></footer>
-  <script type="module" src="js/main.js"></script>
-  <script type="module" src="./js/ownRecipe.mjs"></script>
-</body>
-</html>
```

Comment:
@JosueBCe Remember, you'll not want any `<script>` or `<html>` or `<body>`, etc in these partials. Just the content that you're loading into `#main-content` in `index.html`.